### PR TITLE
Implemented allowNewEmailAccounts

### PR DESCRIPTION
**Pull request by najdanovicivan**
Implemented  allow new email accounts, Reference #125

### CODE REVIEW

**Comment by najdanovicivan** [comment]
Travis CI fails also on master due to deprecation in FUIAccountSettingsOperation. I've mentioned issue here #125

File: FirebaseAuthUI/FUIAuth.h [hunk 1]
```diff
@@ -176,6 +176,11 @@ __attribute__((deprecated("Instead use authUI:didSignInWithAuthDataResult:error:
  */
 @property(nonatomic, assign, getter=isSignInWithEmailHidden) BOOL signInWithEmailHidden;
 
+/** @property allowNewEmailAccounts
+ @brief Whether to allow new user sign, defaults to YES.
+ */
+@property(nonatomic, assign, getter=allowedNewEmailAccounts) BOOL allowNewEmailAccounts;
```
Comment (morganchen12) [inline FirebaseAuthUI/FUIAuth.h]: I don't think this needs a custom getter name (but if you'd like to keep it, consider renaming to `allowsNewEmailAccounts`)

File: FirebaseAuthUI/FUIAuth.m [hunk 2]
```diff
@@ -125,11 +127,20 @@ - (UINavigationController *)authViewController {
...
-      controller = [[FUIEmailEntryViewController alloc] initWithAuthUI:self];
-    }
+      if (self.allowedNewEmailAccounts){
+        if ([self.delegate respondsToSelector:@selector(emailEntryViewControllerForAuthUI:)]) {
+          controller = [self.delegate emailEntryViewControllerForAuthUI:self];
+        } else {
+          controller = [[FUIEmailEntryViewController alloc] initWithAuthUI:self];
+        }
+      }
+      else{
```
Comment (morganchen12) [inline FirebaseAuthUI/FUIAuth.m]: nit: space after `else`

**Comment by najdanovicivan** [comment]
Fixed code nits and merged with master to pass TravisCI

**Pull request merged**